index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <title>Mint Sign Up Page</title>
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css?family=Blinker&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <nav class="header">
+            <div class="header-content">
+                <div id="login-2">
+                    <span id="header-text">Already have an Intuit account?</span>
+                    <br>
+                    <span id="header-link">
+                        <a  href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26adobe_mc%3DMCMID%253D34758894225416457621264800979794955614%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1565729615%26ivid%3D024d5df0-3ed3-47c7-b0f7-a500be1696a9#">I forgot My User Id or Password</a>
+                    </span>
+                </div>
+                <div class="sign-in">
+                    <button type="button" name="sign-in" id="sign-in-button" class="button">
+                        <span class="button-img"></span>
+                            Sign In
+                    </button>
+                </div>
+
+            </div>
+
+        </nav>
+        <main class="body">
+            <div class="brand-name">
+                <a id="brand-img" href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26adobe_mc%3DMCMID%253D34758894225416457621264800979794955614%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1565729615%26ivid%3D024d5df0-3ed3-47c7-b0f7-a500be1696a9#"></a>
+            </div>
+            <div class="add-links">
+                <a href="#"><img class="Color" src="img1.png" alt=""></a>
+            </div>
+            <div class="add-links2">
+                <a href="#"><img class="Color" src="img2.png" alt=""></a>
+            </div>
+            <form class="form" action="#" method="post">
+                <fieldset class="fieldset">
```

Comment:
This fieldset tag is not really needed here, what exactly are we grouping :thinking: